Update high-level.md (#2)
loambuild/loambuild.github.io: src/pages/high-level.md

Command line interface built by the team that maintains the [Soroban CLI](https://github.com/stellar/soroban-cli). Create, build, deploy, and manage both contracts and frontend projects with ease. Get started with `cargo install loam-cli`.

### 3. Smart Contracts

> A package manager for publishing versioned Wasm binaries; a contract registry which lets you deploy contracts and claim a named associated with its contract id; and a contract expiration tracker which will ensures that contracts won't expiration.

Reviewer: ```suggestion
A package manager for publishing versioned Wasm binaries; a contract registry which lets you deploy contracts and claim a named associated with its contract id; and a contract expiration tracker that keeps your data live. 
```
